.vscode/settings.json

```diff
@@ -23,7 +23,6 @@
   },
   "files.insertFinalNewline": true,
   "files.trimTrailingWhitespace": true,
-
```

Comment:
Is the empty line important/necessary here?
It's removed by autosave, but if yes, I can revert the change.

Reply:
No it's not necessary. Probably best to remove it.